sphinx/themes/basic/static/searchtools.js

```diff
@@ -367,8 +372,7 @@ const Search = {
     //Search.lastresults = results.slice();  // a copy
     // console.info("search results:", Search.lastresults);
 
-    // print the results
-    _displayNextItem(results, results.length, searchTerms, highlightTerms);
+    return { results, searchTerms, highlightTerms };
```

Comment:
Refactored this to be seperate from displaying the results, which was necessary to add the test.